### ISSUE TITLE
[OMP] Need Help. admin Catalog page broken after ompfiles deletion and submission database record deletion

### ISSUE THREAD

**Issue by umkuniverse** [pkp/pkp-lib#10532]
## Issue Description
// What is the nature of the issue? How does it impact users?
case background: we facing couple injection issue to our domain powered by OMP 3.4.0-6. exploit/php-shell file injected into our directory and during submission process. we do scanning to our directory and found that illegal-extension file uploaded with .phtml extension into ompfiles directory. we decided to delete the folder with ID 65 inside ompfiles directory. we also do checking inside database and found record inside submission-table with submissionID=65 and delete it (my foul). after this kind of deletion process now we have broken page inside admin > Catalog and any others page was normal. broken page appear blank white page with Catalog title and loading with no css (check with inspect element)
![image](https://github.com/user-attachments/assets/18bbf610-4504-4177-af7e-4f5c6bb29843)

## How to reproduce / Current behavior
// Step-by-step instructions explaining how to reproduce the problem. The environment required, explicit identification of the component in question
error message can reproduce with deleting submission id-folder inside ompfiles and deleting record inside submission-table.

## What application are you using?
 OMP version 3.4.0-6

## Stack used 
- Device: acer swift 3 20GB RAM+1tb nvme
- OS: windows 11
- Browser: google chrome
- Version: 129.0.6668.101

**Comment by asmecher** [comment]
@umkuniverse, next time please start with our support forum (https://forum.pkp.sfu.ca). If an issue winds up being a bug, we'll migrate it back here.

I believe you've encountered #9094. Since you're running a more recent release than 3.4.0-1 (which received the underlying fix), you should be able to resolve it by following the setup directions [here](https://github.com/pkp/pkp-lib/issues/9094#issuecomment-1593690281).

I think the malicious upload is not related to the problem. As long as your OMP files directory is not directly web-accessible, it does not matter if users attempt to upload e.g. `.phtml` files.

**Comment by umkuniverse** [comment]
@asmecher thank you for poin me out to the solution link. we have try with sort order A-Z or Z-A from previous condition is not selected before but still no luck. you may have another suggest to try? we will also go trough the forum to find out the solution. also sorry for posting into wrong place.

**Comment by umkuniverse** [comment]
dear @asmecher, with this message we confirm that the problem solved by your suggestion, and next step go to administration > delete data cache and delete template cache. thank you so much
